features/step_definitions/helpers.rb

```diff
@@ -64,11 +64,11 @@ def initialize_task_params(project, story, user = User.find(:first))
   params
 end
 
-def initialize_work_package_params(project, type = Type.find(:first), parent = nil, user = User.find(:first))
+def initialize_work_package_params(project, type = Type.first, parent = nil, user = User.first)
   params = HashWithIndifferentAccess.new
   params['type_id'] = type.id
   params['parent_id']  = parent.id if parent
-  params['status_id'] = Status.find(:first).id
+  params['status_id'] = Status.first.id
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.